Source: zelda, jtv, Fedora 15
Upstream: yes, Changeset 1722, http://pqxx.org/development/libpqxx/changeset/1722
Reason: error: 'ptrdiff_t' does not name a type

Index: /trunk/include/pqxx/compiler-internal.hxx
===================================================================
--- /trunk/include/pqxx/compiler-internal.hxx	(revision 1704)
+++ /trunk/include/pqxx/compiler-internal.hxx	(revision 1722)
@@ -54,4 +54,6 @@
 #include "pqxx/compiler-public.hxx"
 
+#include <cstddef>
+
 #ifdef PQXX_HAVE_LIMITS
 #include <limits>
Index: /trunk/ChangeLog
===================================================================
--- /trunk/ChangeLog	(revision 1721)
+++ /trunk/ChangeLog	(revision 1722)
@@ -1,2 +1,5 @@
+2011-05-16  Jeroen T. Vermeulen <jtv@xs4all.nl>
+ include/pqxx/compiler-internal.hxx:
+ - Include <cstddef>; may fix #213 on Fedora 15.  Thanks zelda.
 2011-05-10  Jeroen T. Vermeulen <jtv@xs4all.nl>
  tools/pqxx-fulltest:
